Support new option .scancommitforcc

azat opened the pull request:
I don't really sure is it useful for some others.
But I think that it could be, you can just write in commit:
```
Fix memory leak

Cc: tests@product
```
And it will send email to "tests@product" with only this commit.
Thanks.

(And definitely no documentation, but if this will be useful, I will do.)

moy commented:
I rebased your branch, squashed the commits, addressed my comments and added a few tests. I changed the regexp a bit to avoid matching lines starting with Cc and not containing an email.

I pushed the result to master, but feel free to send further fixes.

azat commented:
On Sun, May 24, 2015 at 11:44:31AM -0700, Matthieu Moy wrote:
> I rebased your branch, squashed the commits, addressed my comments and added a few tests. I changed the regexp a bit to avoid matching lines starting with Cc and not containing an email.

Many thanks @moy !

> I pushed the result to master, but feel free to send further fixes.

azat commented:
> I pushed the result to master, but feel free to send further fixes.

One thing that bother me is that you can't use scanCommitForCc without
mailingList, I'm going to send a patch that will fix this.
